tsconfig.json

```
@@ -9,8 +9,10 @@
     "experimentalDecorators": true,
     "removeComments": false,
     "outDir": "dist",
+    "strict": true,
     "allowUnusedLabels": false,
     "noImplicitReturns": true,
+    "strictNullChecks": true,
```

If you use `strict` you don't need `strictNullChecks`. ["Turning this on is equivalent to enabling all of the strict mode family options"](https://www.typescriptlang.org/tsconfig#strict)

```suggestion
    "strict": true,
    "allowUnusedLabels": false,
    "noImplicitReturns": true,
```